index.js

```diff
@@ -0,0 +1,80 @@
+/* eslint-disable */
+document.addEventListener('DOMContentLoaded', () => {
+  // Function Constructor
+  function Book(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+  // Empty Array for Objects!
+  const Books = [];
+  // Function Add for Adding Books;
+  function Add() {
+    const AddButton = document.getElementById('add-btn');
+    // i used as key value, As we know setItem takes two parameters Key and Value
+    let i = 1;
+    // Add Event Listener to Add Button so if any User clicks the Book added as Object!
+    AddButton.addEventListener('click', () => {
+    // To Taking inputs from Users;
+      const Title = document.getElementById('title').value;
+      const Author = document.getElementById('author').value;
+      // The New Object Created with help of 'new' that is constructor func.
+      const book = new Book(Title, Author);
+      // LocalStorage Setup!
+      if (localStorage.length === 0) {
+        localStorage.setItem(i, JSON.stringify(book));
+        // Object book Pushed(Added) in Books Array.
+        Books.push(book);
```

Comment:
Currently, when you add or remove a book, it is updated in the UI👍. But the Books array is not updated with the added or removed book, kindly fix this👍.